cli/cppcheckexecutor.cpp

```diff
@@ -198,76 +198,76 @@ std::size_t GetArrayLength(const T(&)[size])
 
 
 #if defined(USE_UNIX_SIGNAL_HANDLING)
-/*
- * Try to print the callstack.
- * That is very sensitive to the operating system, hardware, compiler and runtime!
- * The code is not meant for production environment, it's using functions not whitelisted for usage in a signal handler function.
- */
+    /*
```

Comment:
It is hard to read the diff when you are making lots of redundant indentation changes like this one. Please remove all redundant indentation changes.